lib/container_server.go

```diff
@@ -129,13 +130,46 @@ func (c *ContainerServer) StorageRuntimeServer() storage.RuntimeServer {
 	return c.storageRuntimeServer
 }
 
+// parseIDMaps makes sense of the four ID mapping options that we might have to
+// process, returning maps for UIDs and GIDs to use as defaults for storage.
+func parseIDMaps(uidMap, gidMap [][3]int, uidMapUser, gidMapGroup string) (uidmap, gidmap []idtools.IDMap, _ error) {
```

Comment:
Same as above, this should be in a common function that can be vendored into podman, buildah and cri-o.